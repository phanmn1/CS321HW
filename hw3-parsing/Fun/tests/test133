// Droppin Parens everywhere! We have an empty
// parenthesized expression as the argument
// to a function call. Who ya gonna call?
int count = 0, z=q,f,f;

int fact(int n, boolean y) {
  fact(());
  count = count + 1;
  if (n<1)
    return 1;
  else
    return n * fact(n-1, z);
}

int x = y;
int y = 1;

void main(boolean f, boolean z, int y) {
  print fact(7,3);
  print fact(x,6,r,parsingIsFun);
  print fact(count,isItThough);
}

