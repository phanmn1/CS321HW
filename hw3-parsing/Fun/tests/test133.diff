1c1
< Invalid syntax at (8,7), (
---
> Invalid syntax at (14,6), n
