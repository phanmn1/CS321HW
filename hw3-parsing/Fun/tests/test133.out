Invalid syntax at (14,6), n
